src/storage-manager.coffee

```diff
@@ -1,20 +1,33 @@
-window.Brainstem ?= {}
+$ = require 'jquery'
+Backbone = require 'backbone'
+inflection = require 'inflection'
 
-# TODO: Record access timestamps on all Brainstem.Models by overloading #get and #set.
+Utils = require './utils'
+Expectation = require './expectation'
+ModelLoader = require './loaders/model-loader'
+CollectionLoader = require './loaders/collection-loader'
+sync = require './sync'
+
+
+# TODO: Record access timestamps on all Models by overloading #get and #set.
 #    - Keep a sorted list (Heap?) of model references
 #    - Clean up the oldest ones if memory is low
 #    - Allow passing a recency parameter to the StorageManager
 
-# The StorageManager class is used to manage a set of Brainstem.Collections.  It is responsible for loading data and
-# maintaining caches.
-class window.Brainstem.StorageManager
+# The StorageManager class is used to manage a set of Collections.
+# It is responsible for loading data and maintaining caches.
+class _StorageManager
```

Comment:
Why the leading underscore in the name? It's definitely not private as we need it in our files and something that we should find a way to publicize so it can be a singleton and standalone.